### DISCUSSION TITLE
Introduce the possibility to disable the "shared across alla languages" warning via config

### DISCUSSION THREAD

**Discussion by fredrikbazooka** [umbraco/Umbraco-CMS#14048] [Features and ideas]
Our editor find the "shared across alla languages" warning in the Umbraco backoffice gui annoying and have asked me if it's possible to prevent this warning from being shown. I find no documentation indicating that this would be possible via config. Is this something that could be introduced?

**Comment by bergmania** [comment]
Hi @fredrikbazooka..

We discussed this internally and do not like the idea of a global setting disabling this, due to the fact you can make unintended changes. In the future this would make good sense as a setting on the singed-in user, but we can't do this right now.

**Comment by fredrikbazooka** [comment]
Hello @bergmania 

Thank you. 

Am I missing something here, or why is this showing. Is it just the fact that the property is the same for all languages or is there something I'm doing wrong, something that changed in newer versions?

As a site developer we carefully choose properties that are shared across languages, so they are supposed to be the same (for example a photo or name of a person), and the editor expects them to be the same. It's never like a random suprise, if so it's done wrong.

So our editors have to click these buttons all day, and they find it annoying that it cannot be turned off. So we still strongly vote for this to be implemented.

Or if possible, if each of the dialogs never would be shown again once you clicked it.

Regards
Fredrik

**Comment by danpikey** [comment]
My client's editors are finding the warning very annoying. There must be a better way of stopping the unwary from changing culture-invariant content. Suggestions:

1. One warning popup on opening the content in question, then just heavily highlighted sections thereafter.
2. Only allow content to be edited when editing the default language. Make it read-only for other langs. 

Thanks in advance for anything that can be done about this.
